src/test/java/com/google/firebase/auth/OidcProviderConfigTest.java

```diff
@@ -51,6 +51,26 @@ public void testJsonDeserialization() throws IOException {
     assertEquals("https://oidc.com/issuer", config.getIssuer());
   }
 
+  @Test
+  public void testCheckSamlProviderId() {
```

Comment:
Is there a better way capture these tests without directly testing on internal methods? May be run tests on public methods `getProviderConfig()` and `deleteProviderConfig()` with invalid arguments to verify this behavior.

Reply:
I've moved the testing to the API level!